### ISSUE TITLE
12net train log

### ISSUE THREAD

**Issue by wuzhiyang2016** [CongWeilin/mtcnn-caffe#34]
## **hai，i have trained pnet，only train classification and bbox regression tasks, here is my train log** 

I0508 11:43:17.349756 124467 solver.cpp:219] Iteration 496000 (470.551 iter/s, 2.12517s/1000 iters), loss = 0.708101
I0508 11:43:17.349814 124467 solver.cpp:238]     Train net output #0: cls_Acc = 0
I0508 11:43:17.349827 124467 solver.cpp:238]     Train net output #1: cls_loss = 0.693147 (* 1 = 0.693147 loss)
I0508 11:43:17.349834 124467 solver.cpp:238]     Train net output #2: roi_loss = 0.0299242 (* 0.5 = 0.0149621 loss)
I0508 11:43:17.349840 124467 sgd_solver.cpp:105] Iteration 496000, lr = 2.81475e-06
I0508 11:43:19.471246 124467 solver.cpp:219] Iteration 497000 (471.378 iter/s, 2.12144s/1000 iters), loss = 0.444717
I0508 11:43:19.471307 124467 solver.cpp:238]     Train net output #0: cls_Acc = 0.8
I0508 11:43:19.471321 124467 solver.cpp:238]     Train net output #1: cls_loss = 0.444725 (* 1 = 0.444725 loss)
I0508 11:43:19.471330 124467 solver.cpp:238]     Train net output #2: roi_loss = 0 (* 0.5 = 0 loss)
I0508 11:43:19.471336 124467 sgd_solver.cpp:105] Iteration 497000, lr = 2.81475e-06
I0508 11:43:21.595211 124467 solver.cpp:219] Iteration 498000 (470.83 iter/s, 2.12391s/1000 iters), loss = 0.185536
I0508 11:43:21.595271 124467 solver.cpp:238]     Train net output #0: cls_Acc = 1
I0508 11:43:21.595288 124467 solver.cpp:238]     Train net output #1: cls_loss = 0.185544 (* 1 = 0.185544 loss)
I0508 11:43:21.595302 124467 solver.cpp:238]     Train net output #2: roi_loss = 0 (* 0.5 = 0 loss)
I0508 11:43:21.595317 124467 sgd_solver.cpp:105] Iteration 498000, lr = 2.81475e-06
I0508 11:43:23.719271 124467 solver.cpp:219] Iteration 499000 (470.809 iter/s, 2.124s/1000 iters), loss = 0.720061
I0508 11:43:23.719334 124467 solver.cpp:238]     Train net output #0: cls_Acc = 0
I0508 11:43:23.719364 124467 solver.cpp:238]     Train net output #1: cls_loss = 0.693147 (* 1 = 0.693147 loss)
I0508 11:43:23.719373 124467 solver.cpp:238]     Train net output #2: roi_loss = 0.0538428 (* 0.5 = 0.0269214 loss)
I0508 11:43:23.719380 124467 sgd_solver.cpp:105] Iteration 499000, lr = 2.81475e-06
I0508 11:43:25.840430 124467 solver.cpp:448] Snapshotting to binary proto file /home/wzy/MTCNN/model/_iter_500000.caffemodel
I0508 11:43:25.842113 124467 sgd_solver.cpp:273] Snapshotting solver state to binary proto file /home/wzy/MTCNN/model/_iter_500000.solverstate
I0508 11:43:25.843174 124467 solver.cpp:311] Iteration 500000, loss = 0.112088
I0508 11:43:25.843196 124467 solver.cpp:316] Optimization Done.
I0508 11:43:25.843201 124467 caffe.cpp:259] Optimization Done.

it look  so strange, and when i test an image, the result  is so bad compare with your model,,,,can you share your train log? @ CongWeilin

**Comment by CongWeilin** [comment]
i can not find it. it has been a long time. your cls_Acc is too low. i remember when training classify task, the accuracy is about 87-94%

**Comment by yungs2017** [comment]
@wuzhiyang2016 how do you trian the pnet? 
when i train it, i always get the NaN loss
